### ISSUE TITLE
Score or Accuracy?

### ISSUE THREAD

**Issue by guilhermevrs** [tariqdaouda/Mariana#17]
Hello @tariqdaouda,

First of all, thank you for this framework. It works very well!

I was wondering how does exactly work every observation made during the training phase.

I mean, supposing I am interested in getting the percentage error of my network on a given test set. What data should I look in this case?

Thank you in advance

**Comment by tariqdaouda** [comment]
Hello @guilhermevrs,

I'm glad that you like it!
The goal of the training is to find parameters that minimize the score. The score is the output of the loss function (also called error function) that measures the distance between the outputs and the targets. It is automatically generated from the cost and the regularizations, and therefore values given by two different models are not necessarily comparable.

The accuracy on the other hand is simply the ratio of well classified examples. So in that case you would want to take (1 - accuracy) to get the ratio of errors and multiply it by 100 to get a percentage.

Cheers,
